idd/Energy+.idd.in

```diff
@@ -35433,7 +35433,6 @@ ZoneHVAC:HybridUnitaryHVAC,
     A1, \field Name
         \required-field
         \type alpha
-        \retaincase
```

Comment:
Yeah I think this is good, I don't know why it was retaincase before.  There shouldn't be anything case specific.